configuration/concepts/WHODrugConcepts.csv

```diff
@@ -0,0 +1,146 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Fully specified name:fr,Short name:fr,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:200

```

Comment:
Why WHODrugConcepts - WHO Drugs will probably have ATC codes, not SNOMED. So maybe change the file name. Also, check with Akhil if such Drug Concepts are generic and/or standard from CIEL dictionary to be bundled with Bahmni. If yes, then we probably dont have to do this using a csv but the metadata zip or sync will take care of it.

Reply:
@angshu updated the file name.

Reply:
@angshu Checked with Akhil, these concepts are standard from CIEL dictionary but for this project we are not using CIEL ZIP. (default-config), so creating them with corresponding mapping codes